client/my-sites/sidebar/body.jsx

```diff
@@ -0,0 +1,62 @@
+import { useSelector } from 'react-redux';
+import SidebarSeparator from 'calypso/layout/sidebar/separator';
+import isSiteWpcomAtomic from 'calypso/state/selectors/is-site-wpcom-atomic';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSidebarIsCollapsed, getSelectedSiteId } from 'calypso/state/ui/selectors';
+import MySitesSidebarUnifiedItem from './item';
+import MySitesSidebarUnifiedMenu from './menu';
+import useSiteMenuItems from './use-site-menu-items';
+import { itemLinkMatches } from './utils';
+import 'calypso/state/admin-menu/init';
+
+import './style.scss';
+
+export const MySitesSidebarUnifiedBody = ( { path, children } ) => {
+	const menuItems = useSiteMenuItems();
+	const sidebarIsCollapsed = useSelector( getSidebarIsCollapsed );
+	const siteId = useSelector( getSelectedSiteId );
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, siteId ) );
+	const isSiteAtomic = useSelector( ( state ) => isSiteWpcomAtomic( state, siteId ) );
+
+	// Jetpack self-hosted sites should open external links to WP Admin in new tabs,
```

Comment:
This comment doesn't really apply as we're about to do the same thing with our untangling. But, not a blocker to change.